[WSL2][python3:x64-linux] build failure

**Host Environment**
- OS: Ubuntu 20.04 LTS through WSL2
- Compiler: -- The C compiler identification is GNU 9.3.0 -- The CXX compiler identification is Clang 10.0.0

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install boost`

**Failure logs** 
```
Computing installation plan...
The following packages will be built and installed:
    boost[core]:x64-linux -> 1.75.0
  * boost-parameter-python[core]:x64-linux -> 1.75.0
  * boost-pfr[core]:x64-linux -> 1.75.0
  * boost-poly-collection[core]:x64-linux -> 1.75.0
  * boost-process[core]:x64-linux -> 1.75.0
  * boost-program-options[core]:x64-linux -> 1.75.0#1
  * boost-python[core]:x64-linux -> 1.75.0#3
  * boost-safe-numerics[core]:x64-linux -> 1.75.0
  * boost-signals2[core]:x64-linux -> 1.75.0
  * boost-sort[core]:x64-linux -> 1.75.0
  * boost-stacktrace[core]:x64-linux -> 1.75.0#1
  * boost-statechart[core]:x64-linux -> 1.75.0
  * boost-static-string[core]:x64-linux -> 1.75.0
  * boost-stl-interfaces[core]:x64-linux -> 1.75.0
  * boost-timer[core]:x64-linux -> 1.75.0#1
  * boost-type-erasure[core]:x64-linux -> 1.75.0#1
  * boost-vmd[core]:x64-linux -> 1.75.0
  * boost-wave[core]:x64-linux -> 1.75.0#1
  * boost-yap[core]:x64-linux -> 1.75.0
  * python3[core]:x64-linux -> 3.9.5
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-linux...
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/bc/bc21f1338507ebd3194af6964c8899c88f8f4bf9.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/db/dba01e8a4b4c044c1ce463e7028792ec5ae0c81e.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/3b/3bf32846d66ed369d4e764e5f953939a1bacb504.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/b3/b338c73261790838e3727309ccbf2dc3acf1cc96.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/3b/3bd70fb2405ebda377f27ac555aef19ddd104944.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/16/16a1645f094924cd40077044f74de79dad12647d.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/23/2339e2f0e6352ae19b30c14321a4324dbdc3a273.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/af/afeab0de983e74cb8b56ff6fee6c80b6dca536e7.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/7e/7e0b0438008033d5f791e2c40ee4a934917f37f4.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/0a/0a8e48e7097158740230ae082ac06f07f5cd1952.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/15/152eab1ba7c89d989fed30106294f4f12147d5e2.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/ba/ba23af3f7defae016817b409c2d4a148f899d140.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/23/237c540b11c3d207e8b301d88be5906c838ac029.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/19/19d62c617df5af69b9ea07f573b7abbfc605ab29.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/87/87c38ff8f4536e8370914f617643a9c3696eb212.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/f2/f23f0d8baafafeb2fa8ce2d12ce051cb797a8831.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/93/93a0d58b3eb3ca0267f36282d985eb30645c87ba.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/e1/e1cc04b2d84402dd1b5a062d7e4866e022fa52a1.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/7d/7dd1557353820f2ec486758c1d901aefb28f2481.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/f9/f99fea0843892dfa58016c8ad0489033c5a03af1.zip
Starting package 1/20: python3:x64-linux
Building package python3[core]:x64-linux...
-- Using cached /home/dominic/dev/vcpkg/downloads/python-cpython-v3.9.5.tar.gz
-- Cleaning sources at /home/dominic/dev/vcpkg/buildtrees/python3/src/v3.9.5-214b854b69.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/dominic/dev/vcpkg/downloads/python-cpython-v3.9.5.tar.gz
-- Applying patch 0001-static-library.patch
-- Applying patch 0002-use-vcpkg-zlib.patch
-- Applying patch 0003-devendor-external-dependencies.patch
-- Applying patch 0004-dont-copy-vcruntime.patch
-- Applying patch 0005-only-build-required-projects.patch
-- Applying patch 0006-fix-duplicate-symbols.patch
-- Using source at /home/dominic/dev/vcpkg/buildtrees/python3/src/v3.9.5-214b854b69.clean
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
-- Installing x64-linux-dbg
-- Building x64-linux-rel
-- Installing x64-linux-rel
CMake Error at ports/python3/portfile.cmake:175 (file):
  file COPY cannot find
  "/home/dominic/dev/vcpkg/packages/python3_x64-linux/bin": No such file or
  directory.
Call Stack (most recent call first):
  scripts/ports.cmake:139 (include)


Error: Building package python3:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: python3:x64-linux
  Vcpkg version: 2021-05-05-unknownhash

Additionally, attach any relevant sections from the log files above.
```
**Additional context**
None

```
CMake Error at ports/python3/portfile.cmake:175 (file):
file COPY cannot find
"/home/dominic/dev/vcpkg/packages/python3_x64-linux/bin": No such file or
directory.
```

i can repro this on fedora 34 with GCC11 libstdc++11
The bin directory did not exist on my machine


[ankurv@trainwavelinux vcpkg.test.20210601]$ ./vcpkg install python3
Computing installation plan...
The following packages will be built and installed:
    python3[core]:x64-linux -> 3.9.5
Detecting compiler hash for triplet x64-linux...
Could not locate cached archive: /home/ankurv/.cache/vcpkg/archives/e8/e81d20de2e844383c4829b3129ba373e15243268.zip
Starting package 1/1: python3:x64-linux
Building package python3[core]:x64-linux...
-- Using cached /home/ankurv/vcpkg.test.20210601/downloads/python-cpython-v3.9.5.tar.gz
-- Cleaning sources at /home/ankurv/vcpkg.test.20210601/buildtrees/python3/src/v3.9.5-214b854b69.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/ankurv/vcpkg.test.20210601/downloads/python-cpython-v3.9.5.tar.gz
-- Applying patch 0001-static-library.patch
-- Applying patch 0002-use-vcpkg-zlib.patch
-- Applying patch 0003-devendor-external-dependencies.patch
-- Applying patch 0004-dont-copy-vcruntime.patch
-- Applying patch 0005-only-build-required-projects.patch
-- Applying patch 0006-fix-duplicate-symbols.patch
-- Using source at /home/ankurv/vcpkg.test.20210601/buildtrees/python3/src/v3.9.5-214b854b69.clean
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
-- Installing x64-linux-dbg
-- Building x64-linux-rel
-- Installing x64-linux-rel
CMake Error at ports/python3/portfile.cmake:175 (file):
  file COPY cannot find
  "/home/ankurv/vcpkg.test.20210601/packages/python3_x64-linux/bin": No such
  file or directory.
Call Stack (most recent call first):
  scripts/ports.cmake:139 (include)

Here's the logs with --debug

[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] BuiltinRegistry initialized with: ""
[DEBUG] Using vcpkg-root: /home/ankurv/vcpkg.test.20210601
[DEBUG] Using installed-root: /home/ankurv/vcpkg.test.20210601/installed
[DEBUG] BuiltinRegistry initialized with: ""
[DEBUG] Using buildtrees-root: /home/ankurv/vcpkg.test.20210601/buildtrees
[DEBUG] Using downloads-root: /home/ankurv/vcpkg.test.20210601/downloads
[DEBUG] Using packages-root: /home/ankurv/vcpkg.test.20210601/packages
[DEBUG] Using scripts-root: /home/ankurv/vcpkg.test.20210601/scripts
[DEBUG] Using ports-root: /home/ankurv/vcpkg.test.20210601/ports
[DEBUG] Using versions-root: /home/ankurv/vcpkg.test.20210601/versions
[DEBUG] Default binary cache path is: /home/ankurv/.cache/vcpkg/archives
Computing installation plan...
[DEBUG] Found path: /usr/bin/cmake
[DEBUG] popen(/home/ankurv/vcpkg.test.20210601/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake --version 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after     4944 us
[DEBUG] popen(/home/ankurv/vcpkg.test.20210601/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake -DVCPKG_ROOT_DIR=/home/ankurv/vcpkg.test.20210601 -DPACKAGES_DIR=/home/ankurv/vcpkg.test.20210601/packages -DBUILDTREES_DIR=/home/ankurv/vcpkg.test.20210601/buildtrees -D_VCPKG_INSTALLED_DIR=/home/ankurv/vcpkg.test.20210601/installed -DDOWNLOADS=/home/ankurv/vcpkg.test.20210601/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/ankurv/vcpkg.test.20210601/buildtrees/0.vcpkg_dep_info.cmake 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after     5611 us
[DEBUG] popen(/home/ankurv/vcpkg.test.20210601/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake -DVCPKG_ROOT_DIR=/home/ankurv/vcpkg.test.20210601 -DPACKAGES_DIR=/home/ankurv/vcpkg.test.20210601/packages -DBUILDTREES_DIR=/home/ankurv/vcpkg.test.20210601/buildtrees -D_VCPKG_INSTALLED_DIR=/home/ankurv/vcpkg.test.20210601/installed -DDOWNLOADS=/home/ankurv/vcpkg.test.20210601/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/ankurv/vcpkg.test.20210601/buildtrees/0.vcpkg_tags.cmake 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after     4672 us
The following packages will be built and installed:
    python3[core]:x64-linux -> 3.9.5
Detecting compiler hash for triplet x64-linux...
[DEBUG] popen(/home/ankurv/vcpkg.test.20210601/downloads/tools/ninja-1.10.1-linux/ninja --version 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after     3682 us
[DEBUG] Found path: /usr/bin/git
[DEBUG] popen(/usr/bin/git --version 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after     3273 us
[DEBUG] popen(/home/ankurv/vcpkg.test.20210601/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake -DCURRENT_PORT_DIR=/home/ankurv/vcpkg.test.20210601/scripts/detect_compiler -DCURRENT_BUILDTREES_DIR=/home/ankurv/vcpkg.test.20210601/buildtrees/detect_compiler -DCURRENT_PACKAGES_DIR=/home/ankurv/vcpkg.test.20210601/packages/detect_compiler_x64-linux -D_HOST_TRIPLET=x64-linux -DCMD=BUILD -DDOWNLOADS=/home/ankurv/vcpkg.test.20210601/downloads -DTARGET_TRIPLET=x64-linux -DTARGET_TRIPLET_FILE=/home/ankurv/vcpkg.test.20210601/triplets/x64-linux.cmake -DVCPKG_BASE_VERSION=2021-05-05 -DVCPKG_CONCURRENCY=25 -DVCPKG_PLATFORM_TOOLSET=external -DGIT=/usr/bin/git -DVCPKG_ROOT_DIR=/home/ankurv/vcpkg.test.20210601 -DPACKAGES_DIR=/home/ankurv/vcpkg.test.20210601/packages -DBUILDTREES_DIR=/home/ankurv/vcpkg.test.20210601/buildtrees -D_VCPKG_INSTALLED_DIR=/home/ankurv/vcpkg.test.20210601/installed -DDOWNLOADS=/home/ankurv/vcpkg.test.20210601/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/ankurv/vcpkg.test.20210601/scripts/ports.cmake 2>&1)
[DEBUG] -- Configuring x64-linux-rel
[DEBUG] -- The C compiler identification is GNU 11.1.1
[DEBUG] -- Detecting C compiler ABI info
[DEBUG] -- Detecting C compiler ABI info - done
[DEBUG] -- Check for working C compiler: /usr/bin/cc - skipped
[DEBUG] -- Detecting C compile features
[DEBUG] -- Detecting C compile features - done
[DEBUG] -- The CXX compiler identification is GNU 11.1.1
[DEBUG] -- Detecting CXX compiler ABI info
[DEBUG] -- Detecting CXX compiler ABI info - done
[DEBUG] -- Check for working CXX compiler: /usr/bin/c++ - skipped
[DEBUG] -- Detecting CXX compile features
[DEBUG] -- Detecting CXX compile features - done
[DEBUG] -- Configuring done
[DEBUG] -- Generating done
[DEBUG] -- Build files have been written to: /home/ankurv/vcpkg.test.20210601/buildtrees/detect_compiler/x64-linux-rel
[DEBUG]
[DEBUG] #COMPILER_HASH#882a3a1e6883bef789761adfe000a8ed202b799d
[DEBUG] #COMPILER_C_HASH#ebfb5f48dd6717540a3d5c1cc26aed06623a0293
[DEBUG] #COMPILER_C_VERSION#11.1.1
[DEBUG] #COMPILER_C_ID#GNU
[DEBUG] #COMPILER_CXX_HASH#9e3a123b9befeb2ae971d60d21f9aa226b4a4272
[DEBUG] #COMPILER_CXX_VERSION#11.1.1
[DEBUG] #COMPILER_CXX_ID#GNU
[DEBUG] CMake Warning:
[DEBUG]   Manually-specified variables were not used by the project:
[DEBUG]
[DEBUG]     BUILD_SHARED_LIBS
[DEBUG]     CMAKE_INSTALL_BINDIR
[DEBUG]     CMAKE_INSTALL_LIBDIR
[DEBUG]     VCPKG_PLATFORM_TOOLSET
[DEBUG]     VCPKG_SET_CHARSET_FLAG
[DEBUG]     _VCPKG_ROOT_DIR
[DEBUG]
[DEBUG]
[DEBUG]
[DEBUG] cmd_execute_and_stream_data() returned 0 after   535419 us
[DEBUG]
[DEBUG] Detected compiler hash for triplet x64-linux: 882a3a1e6883bef789761adfe000a8ed202b799d
[DEBUG] <abientries for python3:x64-linux>
[DEBUG]   0001-static-library.patch|2327732f865a613ae6aa11b2b6926a1ee24c38ec
[DEBUG]   0002-use-vcpkg-zlib.patch|fb81e938ec4afd69f4297ea9f1c24a4702d6b950
[DEBUG]   0003-devendor-external-dependencies.patch|50d4910b50efd1577aa00325bdb7ed9c23138099
[DEBUG]   0004-dont-copy-vcruntime.patch|c34123caa0fad7638bc07432a8fb83b97a30a000
[DEBUG]   0005-only-build-required-projects.patch|09a7fbeddc508fd1da7e74962cd0d76c0428014a
[DEBUG]   0006-fix-duplicate-symbols.patch|4126c3f568c1cfc29e02b105d6d776ba6a1fa0a1
[DEBUG]   0007-restore-support-for-windows-7.patch|dfd7e1394c77b5cc204cf08453ad19624f7c6ef1
[DEBUG]   bzip2|535a6a066906a7fc678854fdd10a753f89707f95
[DEBUG]   cmake|3.20.2
[DEBUG]   expat|219a563eb0b42dc8197dc99805b14fab8933d623
[DEBUG]   features|core
[DEBUG]   libffi|4f3af244f722dda0fa0d599d5910125c90408823
[DEBUG]   liblzma|64fa6ad30db1a7cb52137d14d53068a28ad82e15
[DEBUG]   openssl|b18a026f4468f1cbfe2335b4b8130ef794b49cb1
[DEBUG]   openssl.props.in|89e82b4dfefd4a52447184bab89e945e6a0ccc3a
[DEBUG]   portfile.cmake|ed366a13373f8fa7d62136906724f03ee6ad7504
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   python_vcpkg.props.in|9b5ce8a5de160692071cbf6b44eea65b0b54b07c
[DEBUG]   sqlite3|d90b405309f46694d10c187ebf0431447c25b9d3
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-7094aa317cc64da249e5439fdb7a71dc844ca7a5-882a3a1e6883bef789761adfe000a8ed202b799d
[DEBUG]   usage|8a6e0e3c28040acfb5404da0c253c5c9a6f7dd74
[DEBUG]   vcpkg-cmake-wrapper.cmake|6e4e0ee475ec32a6ba781b5f04611f4d863218e2
[DEBUG]   vcpkg.json|47d0febc7365d79d6d9e64419d267195bc1180d2
[DEBUG]   vcpkg_add_to_path|5b61305ce1f097c857939b0309b008d283fa6658
[DEBUG]   vcpkg_clean_msbuild|6385c359eb42039c03cbc8a3de5eefe38cb7ad63
[DEBUG]   vcpkg_configure_make|e4d27c47bd93b78b9a92cb7c7d1d602eaeafbdb1
[DEBUG]   vcpkg_copy_tool_dependencies|1af6f73afda6fe9536d3e6c593cff97ef3638a88
[DEBUG]   vcpkg_copy_tools|7b4ffeb609a71afb0f1e9653c70ba344c1335d6c
[DEBUG]   vcpkg_execute_required_process|d2520c229d9c3a89d8f512b2cdf6fea381264136
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|a9dd1453b8873c9702731c6c36e79fb0ab6ae486
[DEBUG]   vcpkg_install_make|6b7345be4d3363747f6d75839e87d441025f04d5
[DEBUG]   vcpkg_install_msbuild|67a7a68dbdc6beadd12c0db5e0863a958f2e1202
[DEBUG]   zlib|54a44945e2f097dc0fa935a25952830f5b854a35
[DEBUG] </abientries>
Could not locate cached archive: /home/ankurv/.cache/vcpkg/archives/e8/e81d20de2e844383c4829b3129ba373e15243268.zip
Starting package 1/1: python3:x64-linux
Building package python3[core]:x64-linux...
[DEBUG] popen(/home/ankurv/vcpkg.test.20210601/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake "-DALL_FEATURES=deprecated-win7-support;" -DCURRENT_PORT_DIR=/home/ankurv/vcpkg.test.20210601/ports/python3 -D_HOST_TRIPLET=x64-linux -DFEATURES=core -DPORT=python3 -DVCPKG_USE_HEAD_VERSION=0 -D_VCPKG_DOWNLOAD_TOOL=BUILT_IN -D_VCPKG_EDITABLE=0 -D_VCPKG_NO_DOWNLOADS=0 -DCMD=BUILD -DDOWNLOADS=/home/ankurv/vcpkg.test.20210601/downloads -DTARGET_TRIPLET=x64-linux -DTARGET_TRIPLET_FILE=/home/ankurv/vcpkg.test.20210601/triplets/x64-linux.cmake -DVCPKG_BASE_VERSION=2021-05-05 -DVCPKG_CONCURRENCY=25 -DVCPKG_PLATFORM_TOOLSET=external -DGIT=/usr/bin/git -DVCPKG_ROOT_DIR=/home/ankurv/vcpkg.test.20210601 -DPACKAGES_DIR=/home/ankurv/vcpkg.test.20210601/packages -DBUILDTREES_DIR=/home/ankurv/vcpkg.test.20210601/buildtrees -D_VCPKG_INSTALLED_DIR=/home/ankurv/vcpkg.test.20210601/installed -DDOWNLOADS=/home/ankurv/vcpkg.test.20210601/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/ankurv/vcpkg.test.20210601/scripts/ports.cmake 2>&1)
-- Using cached /home/ankurv/vcpkg.test.20210601/downloads/python-cpython-v3.9.5.tar.gz
-- Cleaning sources at /home/ankurv/vcpkg.test.20210601/buildtrees/python3/src/v3.9.5-214b854b69.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/ankurv/vcpkg.test.20210601/downloads/python-cpython-v3.9.5.tar.gz
-- Applying patch 0001-static-library.patch
-- Applying patch 0002-use-vcpkg-zlib.patch
-- Applying patch 0003-devendor-external-dependencies.patch
-- Applying patch 0004-dont-copy-vcruntime.patch
-- Applying patch 0005-only-build-required-projects.patch
-- Applying patch 0006-fix-duplicate-symbols.patch
-- Using source at /home/ankurv/vcpkg.test.20210601/buildtrees/python3/src/v3.9.5-214b854b69.clean
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
-- Installing x64-linux-dbg
-- Building x64-linux-rel
-- Installing x64-linux-rel
CMake Error at ports/python3/portfile.cmake:175 (file):
  file COPY cannot find
  "/home/ankurv/vcpkg.test.20210601/packages/python3_x64-linux/bin": No such
  file or directory.
Call Stack (most recent call first):
  scripts/ports.cmake:139 (include)


[DEBUG] cmd_execute_and_stream_data() returned 256 after 141338046 us
Error: Building package python3:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: python3:x64-linux
  Vcpkg version: 2021-05-05-unknownhash

Additionally, attach any relevant sections from the log files above.
[DEBUG] /home/ankurv/vcpkg.test.20210601/buildtrees/_vcpkg/src/vcpkg-tool-2021-05-05-9f849c4c43e50d1b16186ae76681c27b0c1be9d9/src/vcpkg/install.cpp(511)
[DEBUG] Exiting after 141982276 us (141979812 us)

@JackBoosY here's the output with the debug flag set:
```bash
dominic@DESKTOP-8IUJULQ:~/dev/vcpkg$ ./vcpkg install boost --debug
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] BuiltinRegistry initialized with: ""
[DEBUG] Using vcpkg-root: /home/dominic/dev/vcpkg
[DEBUG] Using installed-root: /home/dominic/dev/vcpkg/installed
[DEBUG] BuiltinRegistry initialized with: ""
[DEBUG] Using buildtrees-root: /home/dominic/dev/vcpkg/buildtrees
[DEBUG] Using downloads-root: /home/dominic/dev/vcpkg/downloads
[DEBUG] Using packages-root: /home/dominic/dev/vcpkg/packages
[DEBUG] Using scripts-root: /home/dominic/dev/vcpkg/scripts
[DEBUG] Using ports-root: /home/dominic/dev/vcpkg/ports
[DEBUG] Using versions-root: /home/dominic/dev/vcpkg/versions
[DEBUG] Default binary cache path is: /home/dominic/.cache/vcpkg/archives
Computing installation plan...
[DEBUG] Found path: /usr/bin/cmake
[DEBUG] Found path: /bin/cmake
[DEBUG] popen(/home/dominic/dev/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake --version 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after      916 us
[DEBUG] popen(/home/dominic/dev/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake -DVCPKG_ROOT_DIR=/home/dominic/dev/vcpkg -DPACKAGES_DIR=/home/dominic/dev/vcpkg/packages -DBUILDTREES_DIR=/home/dominic/dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=/home/dominic/dev/vcpkg/installed -DDOWNLOADS=/home/dominic/dev/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/dominic/dev/vcpkg/buildtrees/0.vcpkg_dep_info.cmake 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after     1314 us
[DEBUG] popen(/home/dominic/dev/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake -DVCPKG_ROOT_DIR=/home/dominic/dev/vcpkg -DPACKAGES_DIR=/home/dominic/dev/vcpkg/packages -DBUILDTREES_DIR=/home/dominic/dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=/home/dominic/dev/vcpkg/installed -DDOWNLOADS=/home/dominic/dev/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/dominic/dev/vcpkg/buildtrees/1.vcpkg_dep_info.cmake 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after     1462 us
[DEBUG] popen(/home/dominic/dev/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake -DVCPKG_ROOT_DIR=/home/dominic/dev/vcpkg -DPACKAGES_DIR=/home/dominic/dev/vcpkg/packages -DBUILDTREES_DIR=/home/dominic/dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=/home/dominic/dev/vcpkg/installed -DDOWNLOADS=/home/dominic/dev/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/dominic/dev/vcpkg/buildtrees/0.vcpkg_tags.cmake 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after     2047 us
The following packages will be built and installed:
    boost[core]:x64-linux -> 1.75.0
  * boost-parameter-python[core]:x64-linux -> 1.75.0
  * boost-pfr[core]:x64-linux -> 1.75.0
  * boost-poly-collection[core]:x64-linux -> 1.75.0
  * boost-process[core]:x64-linux -> 1.75.0
  * boost-program-options[core]:x64-linux -> 1.75.0#1
  * boost-python[core]:x64-linux -> 1.75.0#3
  * boost-safe-numerics[core]:x64-linux -> 1.75.0
  * boost-signals2[core]:x64-linux -> 1.75.0
  * boost-sort[core]:x64-linux -> 1.75.0
  * boost-stacktrace[core]:x64-linux -> 1.75.0#1
  * boost-statechart[core]:x64-linux -> 1.75.0
  * boost-static-string[core]:x64-linux -> 1.75.0
  * boost-stl-interfaces[core]:x64-linux -> 1.75.0
  * boost-timer[core]:x64-linux -> 1.75.0#1
  * boost-type-erasure[core]:x64-linux -> 1.75.0#1
  * boost-vmd[core]:x64-linux -> 1.75.0
  * boost-wave[core]:x64-linux -> 1.75.0#1
  * boost-yap[core]:x64-linux -> 1.75.0
  * python3[core]:x64-linux -> 3.9.5
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-linux...
[DEBUG] popen(/home/dominic/dev/vcpkg/downloads/tools/ninja-1.10.1-linux/ninja --version 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after      738 us
[DEBUG] Found path: /usr/bin/git
[DEBUG] Found path: /bin/git
[DEBUG] popen(/usr/bin/git --version 2>&1)
[DEBUG] cmd_execute_and_stream_data() returned 0 after      672 us
[DEBUG] popen(/home/dominic/dev/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake -DCURRENT_PORT_DIR=/home/dominic/dev/vcpkg/scripts/detect_compiler -DCURRENT_BUILDTREES_DIR=/home/dominic/dev/vcpkg/buildtrees/detect_compiler -DCURRENT_PACKAGES_DIR=/home/dominic/dev/vcpkg/packages/detect_compiler_x64-linux -D_HOST_TRIPLET=x64-linux -DCMD=BUILD -DDOWNLOADS=/home/dominic/dev/vcpkg/downloads -DTARGET_TRIPLET=x64-linux -DTARGET_TRIPLET_FILE=/home/dominic/dev/vcpkg/triplets/x64-linux.cmake -DVCPKG_BASE_VERSION=2021-05-05 -DVCPKG_CONCURRENCY=17 -DVCPKG_PLATFORM_TOOLSET=external -DGIT=/usr/bin/git -DVCPKG_ROOT_DIR=/home/dominic/dev/vcpkg -DPACKAGES_DIR=/home/dominic/dev/vcpkg/packages -DBUILDTREES_DIR=/home/dominic/dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=/home/dominic/dev/vcpkg/installed -DDOWNLOADS=/home/dominic/dev/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/dominic/dev/vcpkg/scripts/ports.cmake 2>&1)
[DEBUG] -- Configuring x64-linux-rel
[DEBUG] -- The C compiler identification is GNU 9.3.0
[DEBUG] -- Detecting C compiler ABI info
[DEBUG] -- Detecting C compiler ABI info - done
[DEBUG] -- Check for working C compiler: /usr/bin/cc - skipped
[DEBUG] -- Detecting C compile features
[DEBUG] -- Detecting C compile features - done
[DEBUG] -- The CXX compiler identification is Clang 10.0.0
[DEBUG] -- Detecting CXX compiler ABI info
[DEBUG] -- Detecting CXX compiler ABI info - done
[DEBUG] -- Check for working CXX compiler: /usr/bin/c++ - skipped
[DEBUG] -- Detecting CXX compile features
[DEBUG] -- Detecting CXX compile features - done
[DEBUG] -- Configuring done
[DEBUG] -- Generating done
[DEBUG] -- Build files have been written to: /home/dominic/dev/vcpkg/buildtrees/detect_compiler/x64-linux-rel
[DEBUG]
[DEBUG] #COMPILER_HASH#f537c274dd21f214ced727e33e37e05781645f01
[DEBUG] #COMPILER_C_HASH#e89f3b4c21c2005896c964462da4766057dd94e9
[DEBUG] #COMPILER_C_VERSION#9.3.0
[DEBUG] #COMPILER_C_ID#GNU
[DEBUG] #COMPILER_CXX_HASH#e53ba45db9ccea450c1839878212802f03228b07
[DEBUG] #COMPILER_CXX_VERSION#10.0.0
[DEBUG] #COMPILER_CXX_ID#Clang
[DEBUG] CMake Warning:
[DEBUG]   Manually-specified variables were not used by the project:
[DEBUG]
[DEBUG]     BUILD_SHARED_LIBS
[DEBUG]     CMAKE_INSTALL_BINDIR
[DEBUG]     CMAKE_INSTALL_LIBDIR
[DEBUG]     VCPKG_PLATFORM_TOOLSET
[DEBUG]     VCPKG_SET_CHARSET_FLAG
[DEBUG]     _VCPKG_ROOT_DIR
[DEBUG]
[DEBUG]
[DEBUG]
[DEBUG] cmd_execute_and_stream_data() returned 0 after   482407 us
[DEBUG]
[DEBUG] Detected compiler hash for triplet x64-linux: f537c274dd21f214ced727e33e37e05781645f01
[DEBUG] <abientries for python3:x64-linux>
[DEBUG]   0001-static-library.patch|2327732f865a613ae6aa11b2b6926a1ee24c38ec
[DEBUG]   0002-use-vcpkg-zlib.patch|fb81e938ec4afd69f4297ea9f1c24a4702d6b950
[DEBUG]   0003-devendor-external-dependencies.patch|50d4910b50efd1577aa00325bdb7ed9c23138099
[DEBUG]   0004-dont-copy-vcruntime.patch|c34123caa0fad7638bc07432a8fb83b97a30a000
[DEBUG]   0005-only-build-required-projects.patch|09a7fbeddc508fd1da7e74962cd0d76c0428014a
[DEBUG]   0006-fix-duplicate-symbols.patch|4126c3f568c1cfc29e02b105d6d776ba6a1fa0a1
[DEBUG]   0007-restore-support-for-windows-7.patch|dfd7e1394c77b5cc204cf08453ad19624f7c6ef1
[DEBUG]   bzip2|b885fb9cb4a41f20194be206f23a8f056702a517
[DEBUG]   cmake|3.20.2
[DEBUG]   expat|6abb1104f3a72a718992a7a46bcb057246358260
[DEBUG]   features|core
[DEBUG]   libffi|759caaf2cf9b8298b09ffb39f669a08f61757daa
[DEBUG]   liblzma|bb8e6c1876c94d421b5779b81790523350e50f66
[DEBUG]   openssl|9b28eb1e0601cec2aa4fcf15f7569a2b0afbf8a7
[DEBUG]   openssl.props.in|89e82b4dfefd4a52447184bab89e945e6a0ccc3a
[DEBUG]   portfile.cmake|ed366a13373f8fa7d62136906724f03ee6ad7504
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   python_vcpkg.props.in|9b5ce8a5de160692071cbf6b44eea65b0b54b07c
[DEBUG]   sqlite3|cc92d1b994006c962356bbc17019b280b08ebdd9
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   usage|8a6e0e3c28040acfb5404da0c253c5c9a6f7dd74
[DEBUG]   vcpkg-cmake-wrapper.cmake|6e4e0ee475ec32a6ba781b5f04611f4d863218e2
[DEBUG]   vcpkg.json|47d0febc7365d79d6d9e64419d267195bc1180d2
[DEBUG]   vcpkg_add_to_path|5b61305ce1f097c857939b0309b008d283fa6658
[DEBUG]   vcpkg_clean_msbuild|6385c359eb42039c03cbc8a3de5eefe38cb7ad63
[DEBUG]   vcpkg_configure_make|ea3390f3fa3cf6ea8a78c652e11c86751739b740
[DEBUG]   vcpkg_copy_tool_dependencies|1af6f73afda6fe9536d3e6c593cff97ef3638a88
[DEBUG]   vcpkg_copy_tools|7b4ffeb609a71afb0f1e9653c70ba344c1335d6c
[DEBUG]   vcpkg_execute_required_process|d2520c229d9c3a89d8f512b2cdf6fea381264136
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG]   vcpkg_install_make|6b7345be4d3363747f6d75839e87d441025f04d5
[DEBUG]   vcpkg_install_msbuild|67a7a68dbdc6beadd12c0db5e0863a958f2e1202
[DEBUG]   zlib|631b2d7147142dfaaaa80eb690db9eda0dd3f1ed
[DEBUG] </abientries>
[DEBUG] <abientries for boost-python:x64-linux>
[DEBUG]   b2-options.cmake|4314961fa97318ff23ad2b73e94b954409a4298b
[DEBUG]   boost-bind|b4ecfa6a402fc611f21ecc8df8a4b1deec7b8f4c
[DEBUG]   boost-build|4d0c201b70b86642fd7103c76723826462360ede
[DEBUG]   boost-compatibility|798bd613fb5aaf8b2d1df986f22bdd967b3cfb5d
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-conversion|804b82911f1a7954798cbc99903ea37caa147df8
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-detail|09999406226713f7952a7e01f8005ac2a0b513bc
[DEBUG]   boost-foreach|c3dfcb60a3a91cf53aa9d873e5b9273a39a6e1ba
[DEBUG]   boost-function|f85a50664a3acda2daf83e2987a50103338f30f9
[DEBUG]   boost-graph|071bcfd25850e6ed4492180a4242158651b8c67b
[DEBUG]   boost-integer|524eccd83bfffdaaf35ac0fc1ef0b9061f9b469a
[DEBUG]   boost-iterator|52e2a4da829cd1c99a1604195d3518bebfec55cf
[DEBUG]   boost-lexical-cast|d2662ab68b83b9e1a93357d50f10175e7ab6e2fe
[DEBUG]   boost-math|5779df9e991ee9efd8b927248abae98c86f8f485
[DEBUG]   boost-modular-build-helper|ed3fe82ed3e92ab92682a3090ce05d0bf04c8e2e
[DEBUG]   boost-mpl|0a084c16197b43cb338b2f6b034b03484c1e84ee
[DEBUG]   boost-numeric-conversion|34840413cb704690e405617c2790b22e3594229d
[DEBUG]   boost-preprocessor|c60a392e942d59e82af392b4c9f86a26dad4b6e6
[DEBUG]   boost-property-map|9a9c146e70fdb1566b3ad4cedc26f5d3569e3886
[DEBUG]   boost-smart-ptr|645c70cd5464e276cb0d8c99cf9c38e903a17505
[DEBUG]   boost-static-assert|4b29d68605124a8cf7bab4b0e917b8abd89ceedf
[DEBUG]   boost-tuple|7c7465d6e49362d4b3e3922779aea1905f7346fc
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-utility|e395ce4635f7e28d687976d1d64941a660c87438
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|bfce0addc9b8007844fbd684ddcfb51e120f40d0
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   python3|d0e14dc3de84d6cec634c822cf5c1be137105d71
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|49c2c9961a6c3bc62ee85f4d80b3842c66748b0e
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-parameter-python:x64-linux>
[DEBUG]   boost-mpl|0a084c16197b43cb338b2f6b034b03484c1e84ee
[DEBUG]   boost-parameter|d21ef0026ceb893aa8561d17edc30a49548439c0
[DEBUG]   boost-preprocessor|c60a392e942d59e82af392b4c9f86a26dad4b6e6
[DEBUG]   boost-python|16bb73509765dee82e1d30958bca378660bc4fdb
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|d37c193e58a8b0e8b9c47d6344d10aee1650d94a
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|d20d48e939cc207bb99935de966cfaeaa89c29a9
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-pfr:x64-linux>
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|1772f9948f67cbc8f7f2ed6ce4382354d895e289
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|92b41d856c89dd0e7f5bf05ceae64e233a70b3f7
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-vmd:x64-linux>
[DEBUG]   boost-preprocessor|c60a392e942d59e82af392b4c9f86a26dad4b6e6
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|1f5ee3640b8f1f543a696b79ab05fd60a88204c0
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|c467873b69137ba613afea31c76616d121ce47b9
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-type-erasure:x64-linux>
[DEBUG]   boost-assert|fffe8b963433c368562c4e5c9e178354c48383a0
[DEBUG]   boost-build|4d0c201b70b86642fd7103c76723826462360ede
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-detail|09999406226713f7952a7e01f8005ac2a0b513bc
[DEBUG]   boost-fusion|ae078c9ffd0293a9f31fad8f05d271a2c37395e7
[DEBUG]   boost-iterator|52e2a4da829cd1c99a1604195d3518bebfec55cf
[DEBUG]   boost-modular-build-helper|ed3fe82ed3e92ab92682a3090ce05d0bf04c8e2e
[DEBUG]   boost-mp11|108c178f78c283268a974538bae2d8b3d66bea8e
[DEBUG]   boost-mpl|0a084c16197b43cb338b2f6b034b03484c1e84ee
[DEBUG]   boost-preprocessor|c60a392e942d59e82af392b4c9f86a26dad4b6e6
[DEBUG]   boost-smart-ptr|645c70cd5464e276cb0d8c99cf9c38e903a17505
[DEBUG]   boost-thread|95e14b9b67b2c3f04be3c0b5d6aa9b03bff10ae7
[DEBUG]   boost-throw-exception|01dab0dbff3107bc85321eb1ed985b48a408d10e
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-typeof|da5041f67285fadd2052d4138c0aecb56a01b8d5
[DEBUG]   boost-utility|e395ce4635f7e28d687976d1d64941a660c87438
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   boost-vmd|963e509355066f1f22975bb1eb5bcc77026f5263
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|a5ad93b42167843ba4d4ffbe6be4d6f782c08da2
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|4276ff39fcff15bbcde367725d3ef9bbb584b609
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-poly-collection:x64-linux>
[DEBUG]   boost-assert|fffe8b963433c368562c4e5c9e178354c48383a0
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-detail|09999406226713f7952a7e01f8005ac2a0b513bc
[DEBUG]   boost-iterator|52e2a4da829cd1c99a1604195d3518bebfec55cf
[DEBUG]   boost-mp11|108c178f78c283268a974538bae2d8b3d66bea8e
[DEBUG]   boost-mpl|0a084c16197b43cb338b2f6b034b03484c1e84ee
[DEBUG]   boost-type-erasure|7f2e684e9e10b7062573227612adbb5de70ab904
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|3819f7d048e729d3570b9f93177cc59835ea56ce
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|9093afb84083c398a0b1b784fd4771ce19685c38
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-process:x64-linux>
[DEBUG]   boost-algorithm|4ae0dd8e449af0f4d2afc24dde459cb65737be4b
[DEBUG]   boost-asio|c2a9f268a868788e7d368a6be0ab12310eee2dcb
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-filesystem|93242645644824d2bdeb1ffc6bf6e66d9196c519
[DEBUG]   boost-fusion|ae078c9ffd0293a9f31fad8f05d271a2c37395e7
[DEBUG]   boost-iterator|52e2a4da829cd1c99a1604195d3518bebfec55cf
[DEBUG]   boost-move|5e7faa2df07991b5524172d924882284a358f0ed
[DEBUG]   boost-optional|bf002bfcd9a17c307124161e71aea9d2d1617b19
[DEBUG]   boost-system|c01021a75868ae6f2cbba6c19ce50f747de16ece
[DEBUG]   boost-tokenizer|a01f8bcf66233af6a684f0f0bf84ed0d2d926b6b
[DEBUG]   boost-type-index|16025aa69f9b735d5e6ebfce683b7692315b4108
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   boost-winapi|cbb8fcff6c02a2d0241498c42f1d667b6e3943ad
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|da5742964c0312fe31588c94d156135fcfda1cef
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|eaa1ca55a15bbd6e06f3a38fcffcfb13a0902e83
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-program-options:x64-linux>
[DEBUG]   boost-any|31bac74049ce34f296a4ea7b86aeb6230d5b5f18
[DEBUG]   boost-bind|b4ecfa6a402fc611f21ecc8df8a4b1deec7b8f4c
[DEBUG]   boost-build|4d0c201b70b86642fd7103c76723826462360ede
[DEBUG]   boost-compatibility|798bd613fb5aaf8b2d1df986f22bdd967b3cfb5d
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-detail|09999406226713f7952a7e01f8005ac2a0b513bc
[DEBUG]   boost-function|f85a50664a3acda2daf83e2987a50103338f30f9
[DEBUG]   boost-iterator|52e2a4da829cd1c99a1604195d3518bebfec55cf
[DEBUG]   boost-lexical-cast|d2662ab68b83b9e1a93357d50f10175e7ab6e2fe
[DEBUG]   boost-math|5779df9e991ee9efd8b927248abae98c86f8f485
[DEBUG]   boost-modular-build-helper|ed3fe82ed3e92ab92682a3090ce05d0bf04c8e2e
[DEBUG]   boost-smart-ptr|645c70cd5464e276cb0d8c99cf9c38e903a17505
[DEBUG]   boost-static-assert|4b29d68605124a8cf7bab4b0e917b8abd89ceedf
[DEBUG]   boost-throw-exception|01dab0dbff3107bc85321eb1ed985b48a408d10e
[DEBUG]   boost-tokenizer|a01f8bcf66233af6a684f0f0bf84ed0d2d926b6b
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|ed68e70ad2930fd81187e073176ab7bf891e6e3c
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|186bf056dd8682156826471a7dcfe641ffbb1336
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-safe-numerics:x64-linux>
[DEBUG]   boost-concept-check|a83ed432c5a7ed543b474273ffb395156936d0ec
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-integer|524eccd83bfffdaaf35ac0fc1ef0b9061f9b469a
[DEBUG]   boost-logic|718446555a6559328bfa6437833313b63f69e6ea
[DEBUG]   boost-mp11|108c178f78c283268a974538bae2d8b3d66bea8e
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|f2b00e8bea0e9e807515371a7a6ba399c0948ffb
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|fcb6bde22e8f9b7cc5ec73c6bf00e75abd4a7d7f
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-signals2:x64-linux>
[DEBUG]   boost-assert|fffe8b963433c368562c4e5c9e178354c48383a0
[DEBUG]   boost-bind|b4ecfa6a402fc611f21ecc8df8a4b1deec7b8f4c
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-detail|09999406226713f7952a7e01f8005ac2a0b513bc
[DEBUG]   boost-function|f85a50664a3acda2daf83e2987a50103338f30f9
[DEBUG]   boost-iterator|52e2a4da829cd1c99a1604195d3518bebfec55cf
[DEBUG]   boost-mpl|0a084c16197b43cb338b2f6b034b03484c1e84ee
[DEBUG]   boost-optional|bf002bfcd9a17c307124161e71aea9d2d1617b19
[DEBUG]   boost-parameter|d21ef0026ceb893aa8561d17edc30a49548439c0
[DEBUG]   boost-predef|c1616bdfd9be2852b5dd82ad10bc0ab93a3a1632
[DEBUG]   boost-preprocessor|c60a392e942d59e82af392b4c9f86a26dad4b6e6
[DEBUG]   boost-smart-ptr|645c70cd5464e276cb0d8c99cf9c38e903a17505
[DEBUG]   boost-throw-exception|01dab0dbff3107bc85321eb1ed985b48a408d10e
[DEBUG]   boost-tuple|7c7465d6e49362d4b3e3922779aea1905f7346fc
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-utility|e395ce4635f7e28d687976d1d64941a660c87438
[DEBUG]   boost-variant|c06a032f4afc0af7874c05fe808b3c42ecd1b86e
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|b201239d023578bfa84c89c263cd06598d3ad9eb
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|05d8aec63c909e87845a9da6813466be298eb0e4
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-sort:x64-linux>
[DEBUG]   boost-integer|524eccd83bfffdaaf35ac0fc1ef0b9061f9b469a
[DEBUG]   boost-range|22bfcd4355ac1cd7c61e619083cfa13958330f26
[DEBUG]   boost-static-assert|4b29d68605124a8cf7bab4b0e917b8abd89ceedf
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-utility|e395ce4635f7e28d687976d1d64941a660c87438
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|13304938282cf501a439aec0b2364afac55537ed
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|3aaeffdf1946f091f061f54d939ca6a0e216fcfa
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-stacktrace:x64-linux>
[DEBUG]   boost-array|0ff1ce591c9d7c347c8dd798eacb4915571fc85a
[DEBUG]   boost-build|4d0c201b70b86642fd7103c76723826462360ede
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-container-hash|9fa36f7e3d34dcf9b0dbdb4030eece161934b9d8
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-modular-build-helper|ed3fe82ed3e92ab92682a3090ce05d0bf04c8e2e
[DEBUG]   boost-predef|c1616bdfd9be2852b5dd82ad10bc0ab93a3a1632
[DEBUG]   boost-static-assert|4b29d68605124a8cf7bab4b0e917b8abd89ceedf
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   boost-winapi|cbb8fcff6c02a2d0241498c42f1d667b6e3943ad
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|5c3037abf60bc3bbacc3fe8a9f0d25ffc8c2c6f0
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|a157d1a5d2d17f48be7685c25e17420ad81a8ab2
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-statechart:x64-linux>
[DEBUG]   boost-assert|fffe8b963433c368562c4e5c9e178354c48383a0
[DEBUG]   boost-bind|b4ecfa6a402fc611f21ecc8df8a4b1deec7b8f4c
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-conversion|804b82911f1a7954798cbc99903ea37caa147df8
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-detail|09999406226713f7952a7e01f8005ac2a0b513bc
[DEBUG]   boost-function|f85a50664a3acda2daf83e2987a50103338f30f9
[DEBUG]   boost-mpl|0a084c16197b43cb338b2f6b034b03484c1e84ee
[DEBUG]   boost-smart-ptr|645c70cd5464e276cb0d8c99cf9c38e903a17505
[DEBUG]   boost-static-assert|4b29d68605124a8cf7bab4b0e917b8abd89ceedf
[DEBUG]   boost-thread|95e14b9b67b2c3f04be3c0b5d6aa9b03bff10ae7
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|aca3fcb16c89975b3e64a702bead14b07dac2157
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|7ebd3085a820022159e18cd3960cbfdf8bb05c50
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-static-string:x64-linux>
[DEBUG]   boost-assert|fffe8b963433c368562c4e5c9e178354c48383a0
[DEBUG]   boost-container-hash|9fa36f7e3d34dcf9b0dbdb4030eece161934b9d8
[DEBUG]   boost-static-assert|4b29d68605124a8cf7bab4b0e917b8abd89ceedf
[DEBUG]   boost-throw-exception|01dab0dbff3107bc85321eb1ed985b48a408d10e
[DEBUG]   boost-utility|e395ce4635f7e28d687976d1d64941a660c87438
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|928f5dec8c9187068e31bba3478a18430a73081b
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|bbb98c7c53374ef3b7a61d07ebe80c5cff248960
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-stl-interfaces:x64-linux>
[DEBUG]   boost-assert|fffe8b963433c368562c4e5c9e178354c48383a0
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|86770095e244510640a2749a5d754ee59c3888dd
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|5e28af510fbf107061dce601576f0af6eea02710
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-timer:x64-linux>
[DEBUG]   boost-build|4d0c201b70b86642fd7103c76723826462360ede
[DEBUG]   boost-chrono|572d3b6c3c76f2c4a1b63b895cbfc1cd71bae912
[DEBUG]   boost-compatibility|798bd613fb5aaf8b2d1df986f22bdd967b3cfb5d
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-integer|524eccd83bfffdaaf35ac0fc1ef0b9061f9b469a
[DEBUG]   boost-io|d7c1dbea9854c2ee0b62c11f8d4eb4a0bf365ee6
[DEBUG]   boost-modular-build-helper|ed3fe82ed3e92ab92682a3090ce05d0bf04c8e2e
[DEBUG]   boost-predef|c1616bdfd9be2852b5dd82ad10bc0ab93a3a1632
[DEBUG]   boost-system|c01021a75868ae6f2cbba6c19ce50f747de16ece
[DEBUG]   boost-throw-exception|01dab0dbff3107bc85321eb1ed985b48a408d10e
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|287e8bf5a5ea26ee493ee4090a7f7a9b3eff461b
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|83dfa7ab2c9870174c5bb1761b938cc104d8825e
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-wave:x64-linux>
[DEBUG]   boost-assert|fffe8b963433c368562c4e5c9e178354c48383a0
[DEBUG]   boost-build|4d0c201b70b86642fd7103c76723826462360ede
[DEBUG]   boost-concept-check|a83ed432c5a7ed543b474273ffb395156936d0ec
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-detail|09999406226713f7952a7e01f8005ac2a0b513bc
[DEBUG]   boost-filesystem|93242645644824d2bdeb1ffc6bf6e66d9196c519
[DEBUG]   boost-integer|524eccd83bfffdaaf35ac0fc1ef0b9061f9b469a
[DEBUG]   boost-iterator|52e2a4da829cd1c99a1604195d3518bebfec55cf
[DEBUG]   boost-lexical-cast|d2662ab68b83b9e1a93357d50f10175e7ab6e2fe
[DEBUG]   boost-math|5779df9e991ee9efd8b927248abae98c86f8f485
[DEBUG]   boost-modular-build-helper|ed3fe82ed3e92ab92682a3090ce05d0bf04c8e2e
[DEBUG]   boost-mpl|0a084c16197b43cb338b2f6b034b03484c1e84ee
[DEBUG]   boost-multi-index|3dec974f68f712f5c4d2e528cc1f8abd465d7309
[DEBUG]   boost-optional|bf002bfcd9a17c307124161e71aea9d2d1617b19
[DEBUG]   boost-pool|00e6bf605e66d5d0188024c7ae84ddb9d6a4d948
[DEBUG]   boost-preprocessor|c60a392e942d59e82af392b4c9f86a26dad4b6e6
[DEBUG]   boost-serialization|a47db4609a1bfaa6f344b992d1a1e887cf953daf
[DEBUG]   boost-smart-ptr|645c70cd5464e276cb0d8c99cf9c38e903a17505
[DEBUG]   boost-spirit|30493ea4b9963d10ec29917c37a10ca8b41fdb9a
[DEBUG]   boost-static-assert|4b29d68605124a8cf7bab4b0e917b8abd89ceedf
[DEBUG]   boost-throw-exception|01dab0dbff3107bc85321eb1ed985b48a408d10e
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|04a8d24b81c9f75b32f548197d5c15aca757ba28
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|d55c76f961ab3f1fe15267e6a25084fef9133578
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost-yap:x64-linux>
[DEBUG]   boost-hana|7867db414b1e8e2418b37bdab54fe1e0a8f97b6c
[DEBUG]   boost-preprocessor|c60a392e942d59e82af392b4c9f86a26dad4b6e6
[DEBUG]   boost-type-index|16025aa69f9b735d5e6ebfce683b7692315b4108
[DEBUG]   boost-vcpkg-helpers|718bfb219c70b84a0f66e20ae20e4568ad674708
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|34b85ebe3e4436f222a9ec0431de629b59ceea36
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   vcpkg.json|b22f47ecb51656b5e558e2ecdf9b3713adc0448b
[DEBUG]   vcpkg_from_git|99e83c6ffac2dd07afaf73d795988ca5dec2dc8b
[DEBUG]   vcpkg_from_github|32e0b058977360f38e0582cc64acbe0780dcfd6a
[DEBUG] </abientries>
[DEBUG] <abientries for boost:x64-linux>
[DEBUG]   0001-Fix-boost-ICU-support.patch|a2cd0fd0de05b59016b20030dcfe03f77b38ac2c
[DEBUG]   boost-accumulators|38e34f09aa469fd5bae760fc2c6f61689d124e46
[DEBUG]   boost-algorithm|4ae0dd8e449af0f4d2afc24dde459cb65737be4b
[DEBUG]   boost-align|5606a287499fbee5482f04a64e11e06365864dcb
[DEBUG]   boost-any|31bac74049ce34f296a4ea7b86aeb6230d5b5f18
[DEBUG]   boost-array|0ff1ce591c9d7c347c8dd798eacb4915571fc85a
[DEBUG]   boost-asio|c2a9f268a868788e7d368a6be0ab12310eee2dcb
[DEBUG]   boost-assert|fffe8b963433c368562c4e5c9e178354c48383a0
[DEBUG]   boost-assign|af64c93348fe3a380af46164d607ce970c5faf2d
[DEBUG]   boost-atomic|2be348434889e00d40d4811d29c342b1effea270
[DEBUG]   boost-beast|002b9da9ad194c4f75fa1d3de10ad3cdf6fb1fba
[DEBUG]   boost-bimap|6e69652f013c446f1770fe1a1edd85a2e8431b6d
[DEBUG]   boost-bind|b4ecfa6a402fc611f21ecc8df8a4b1deec7b8f4c
[DEBUG]   boost-callable-traits|0f13f449eeb58472f5a95394a1ece25fcec67d34
[DEBUG]   boost-chrono|572d3b6c3c76f2c4a1b63b895cbfc1cd71bae912
[DEBUG]   boost-circular-buffer|a75e1b124836c67497523c7ee5bcd3e8d420291d
[DEBUG]   boost-compatibility|798bd613fb5aaf8b2d1df986f22bdd967b3cfb5d
[DEBUG]   boost-compute|d6ac31c93374bdcab6799668b9439da0438281f1
[DEBUG]   boost-concept-check|a83ed432c5a7ed543b474273ffb395156936d0ec
[DEBUG]   boost-config|f669702b822b5da6b9bd433342dece69f131ce37
[DEBUG]   boost-container|a0ba7a0360c2b8f4e2e455b4ee7fa006b4c6aa66
[DEBUG]   boost-container-hash|9fa36f7e3d34dcf9b0dbdb4030eece161934b9d8
[DEBUG]   boost-context|6a7b5f0a4f27345874c54ab22554067deb0ed5e6
[DEBUG]   boost-contract|643cce4ca3e6c4be6747ff0c54047c4198712506
[DEBUG]   boost-conversion|804b82911f1a7954798cbc99903ea37caa147df8
[DEBUG]   boost-convert|3387166f66963b6824df74bcc49a1d4766917ad2
[DEBUG]   boost-core|38106b0f6892b3fb8046b34dc988ac155ef30a59
[DEBUG]   boost-coroutine|bfef1195e556eceea384ece117362c0aaefd10d0
[DEBUG]   boost-coroutine2|6f9d0fe75340870c3441dbdcc82026e04f2588ab
[DEBUG]   boost-crc|74513007ece529aacd644dbceb9c4ed014f73191
[DEBUG]   boost-date-time|105e0b72dcc483e723bf475b1de8dd27d2d26a10
[DEBUG]   boost-detail|09999406226713f7952a7e01f8005ac2a0b513bc
[DEBUG]   boost-dll|0600657c6845fac7abbe13ffe0af3690dfecdd91
[DEBUG]   boost-dynamic-bitset|96bc02a31b9941a99527461f09dcee904a0d769e
[DEBUG]   boost-endian|9679709ed917c3c4c18bbc37ef2bc43525fa2250
[DEBUG]   boost-exception|72b6f6764d7669612fc10d9dbc777ef494d08fdf
[DEBUG]   boost-fiber|386431ebc6f916f240733c75c7ebeba150a3633e
[DEBUG]   boost-filesystem|93242645644824d2bdeb1ffc6bf6e66d9196c519
[DEBUG]   boost-flyweight|4efaa05c0ed624440628884ba7aa0bdf77aeb583
[DEBUG]   boost-foreach|c3dfcb60a3a91cf53aa9d873e5b9273a39a6e1ba
[DEBUG]   boost-format|08aeffde0ea2b5d543bbef841d770a734bf8ce44
[DEBUG]   boost-function|f85a50664a3acda2daf83e2987a50103338f30f9
[DEBUG]   boost-function-types|175e911e701ecd0bdbd07b865ef1fe1eaf9291a3
[DEBUG]   boost-functional|e6a8714787df7604f44d54f14b4b1259e5ff6a4a
[DEBUG]   boost-fusion|ae078c9ffd0293a9f31fad8f05d271a2c37395e7
[DEBUG]   boost-geometry|10a850b30aa656e717d4d2891cbeba6b451df327
[DEBUG]   boost-gil|445979da65755c632f3e45898cc11d53887845d5
[DEBUG]   boost-graph|071bcfd25850e6ed4492180a4242158651b8c67b
[DEBUG]   boost-graph-parallel|de358a4ab74954c34264acb111cf3533f6286048
[DEBUG]   boost-hana|7867db414b1e8e2418b37bdab54fe1e0a8f97b6c
[DEBUG]   boost-heap|3f6b48f72ac89b7a4a1aabceb961902e4aac7395
[DEBUG]   boost-histogram|c4d3142c9841bcceb83d9df9628da376a9f3dbf4
[DEBUG]   boost-hof|158184b37dd0ed6975337ef5cff215ecb60020c8
[DEBUG]   boost-icl|fbde3956eb860164a5894273a4dc84a90195bb60
[DEBUG]   boost-integer|524eccd83bfffdaaf35ac0fc1ef0b9061f9b469a
[DEBUG]   boost-interprocess|f2de28d4bcec950f8985be2c2dc8e0caa7f7063e
[DEBUG]   boost-interval|57c6c05088774273e6e27cd295ded63d2a1669fa
[DEBUG]   boost-intrusive|7ed58592075e026f950b5665e59d1625b65aec95
[DEBUG]   boost-io|d7c1dbea9854c2ee0b62c11f8d4eb4a0bf365ee6
[DEBUG]   boost-iostreams|2da75e8993e79cd7704cad9e25764b7a7b8bda85
[DEBUG]   boost-iterator|52e2a4da829cd1c99a1604195d3518bebfec55cf
[DEBUG]   boost-json|d73b46068b6e3c7dd775cb48a01c1613e78a5130
[DEBUG]   boost-lambda|eaa4bbfdf6e2c18642634e7e5b8be5bf67f87eff
[DEBUG]   boost-leaf|34a10a73599aab36b75a2a97c1fb251f83bb53ac
[DEBUG]   boost-lexical-cast|d2662ab68b83b9e1a93357d50f10175e7ab6e2fe
[DEBUG]   boost-local-function|e4f6e74d4a83735d753ef58b8421fb29ec8304cc
[DEBUG]   boost-locale|0cdb4fc11fabeace2039a175341af1ee25b4dace
[DEBUG]   boost-lockfree|71cc23609b618cd837c11fb490f52b3e1aa0d84b
[DEBUG]   boost-log|ee4de22c01e3e7a9e6230c2c75fda7653dbf0c8b
[DEBUG]   boost-logic|718446555a6559328bfa6437833313b63f69e6ea
[DEBUG]   boost-math|5779df9e991ee9efd8b927248abae98c86f8f485
[DEBUG]   boost-metaparse|c178cb8d38bc678016c907aa1c3ade03a6749c90
[DEBUG]   boost-move|5e7faa2df07991b5524172d924882284a358f0ed
[DEBUG]   boost-mp11|108c178f78c283268a974538bae2d8b3d66bea8e
[DEBUG]   boost-mpl|0a084c16197b43cb338b2f6b034b03484c1e84ee
[DEBUG]   boost-msm|379d53d14f4123429b46efe1261276355dcf9fcb
[DEBUG]   boost-multi-array|2c21d2d0f2c5ce75cabdec9bbdd6d23d728488ec
[DEBUG]   boost-multi-index|3dec974f68f712f5c4d2e528cc1f8abd465d7309
[DEBUG]   boost-multiprecision|a30231c3216e974c041d66fed9a9c4fe7ee575b4
[DEBUG]   boost-nowide|bf57df081c3466f013e120ace7a8006abae58f45
[DEBUG]   boost-numeric-conversion|34840413cb704690e405617c2790b22e3594229d
[DEBUG]   boost-odeint|0895547f303668eec01b2e50fef346c13a2bba81
[DEBUG]   boost-optional|bf002bfcd9a17c307124161e71aea9d2d1617b19
[DEBUG]   boost-outcome|bf0bced54d89ce115131f9ad3fe0f01d212cd83a
[DEBUG]   boost-parameter|d21ef0026ceb893aa8561d17edc30a49548439c0
[DEBUG]   boost-parameter-python|ffbf0331ccfff2f5d79fa580d871a442c3d886c0
[DEBUG]   boost-pfr|cc82c971d17bf22dd8a44f68dbcee4b46c5f5bd7
[DEBUG]   boost-phoenix|072ae78076898c5b653fe4e4d99d6da3de4f39e2
[DEBUG]   boost-poly-collection|931432a6328946a29c9d97d0d6f92faf89eb34b2
[DEBUG]   boost-polygon|ce60a3a31d85e287621c5c31a1c239892699a713
[DEBUG]   boost-pool|00e6bf605e66d5d0188024c7ae84ddb9d6a4d948
[DEBUG]   boost-predef|c1616bdfd9be2852b5dd82ad10bc0ab93a3a1632
[DEBUG]   boost-preprocessor|c60a392e942d59e82af392b4c9f86a26dad4b6e6
[DEBUG]   boost-process|d10ca8bac6d50c17c352f39af91897f339c66938
[DEBUG]   boost-program-options|de06cd55f5f3c3008d380e8e978ca4f0d5ed54db
[DEBUG]   boost-property-map|9a9c146e70fdb1566b3ad4cedc26f5d3569e3886
[DEBUG]   boost-property-tree|905d853cc1f6fbc2057c02ff3c7ea462b45b0f99
[DEBUG]   boost-proto|d4ee293aa090f200f38bcc351f97f881866c3c59
[DEBUG]   boost-ptr-container|e11d7e52d3241da085661f731b6fd42f1f448f4c
[DEBUG]   boost-python|16bb73509765dee82e1d30958bca378660bc4fdb
[DEBUG]   boost-qvm|a84d3be8f41fc258292e226635be3cbe231a742b
[DEBUG]   boost-random|9b5c72f1502a23c60c49cbabdd32e12c34b03612
[DEBUG]   boost-range|22bfcd4355ac1cd7c61e619083cfa13958330f26
[DEBUG]   boost-ratio|800447d2fcf16d618293769a8082e9bca9c64b0a
[DEBUG]   boost-rational|d7cb5b4c1867976961e6c4ac4f1e60ed16f9f81a
[DEBUG]   boost-regex|0b9da912ae1f910eaed895605999f757b43ec08c
[DEBUG]   boost-safe-numerics|da74a16a2c28175d0f2155233c8a9134e5474fbe
[DEBUG]   boost-scope-exit|e4c042cabfa7ff008f136d5728d56c2e0c9b9c00
[DEBUG]   boost-serialization|a47db4609a1bfaa6f344b992d1a1e887cf953daf
[DEBUG]   boost-signals2|23bbc0ec8e179da2aa96e6f1aab0b13ad3b61574
[DEBUG]   boost-smart-ptr|645c70cd5464e276cb0d8c99cf9c38e903a17505
[DEBUG]   boost-sort|f0f8872146029e7ef5edaac4129c5a2a5d616cb4
[DEBUG]   boost-spirit|30493ea4b9963d10ec29917c37a10ca8b41fdb9a
[DEBUG]   boost-stacktrace|ae96a6214dfb68c9e1c7bda46b823cf925668f0f
[DEBUG]   boost-statechart|3da9a0cd1e898ad6a4dcb05025af248dd388ae95
[DEBUG]   boost-static-assert|4b29d68605124a8cf7bab4b0e917b8abd89ceedf
[DEBUG]   boost-static-string|fec11d42c649937a9250343aca3f42a3911f44e0
[DEBUG]   boost-stl-interfaces|27e7202ea9680bfa6bef1d4b18d6a71d844735a7
[DEBUG]   boost-system|c01021a75868ae6f2cbba6c19ce50f747de16ece
[DEBUG]   boost-test|3909ac6c9d85c79c309ec0930d201228e6c26c47
[DEBUG]   boost-thread|95e14b9b67b2c3f04be3c0b5d6aa9b03bff10ae7
[DEBUG]   boost-throw-exception|01dab0dbff3107bc85321eb1ed985b48a408d10e
[DEBUG]   boost-timer|139c3cfbdbc09f2a2931130e30195d6e79a9d514
[DEBUG]   boost-tokenizer|a01f8bcf66233af6a684f0f0bf84ed0d2d926b6b
[DEBUG]   boost-tti|ca889f9faa3704f6f9bd19b423c8d160188f954d
[DEBUG]   boost-tuple|7c7465d6e49362d4b3e3922779aea1905f7346fc
[DEBUG]   boost-type-erasure|7f2e684e9e10b7062573227612adbb5de70ab904
[DEBUG]   boost-type-index|16025aa69f9b735d5e6ebfce683b7692315b4108
[DEBUG]   boost-type-traits|9a048dde27b5f4482076729f9a57ca3e2585eb1e
[DEBUG]   boost-typeof|da5041f67285fadd2052d4138c0aecb56a01b8d5
[DEBUG]   boost-ublas|0b7879e64e5e1c264c36dcd0cd6cea742397c261
[DEBUG]   boost-units|1ecf88c63e13bcf7bd51db1b01375f7cd5f8864b
[DEBUG]   boost-unordered|68619aede1228d28890d85f77ba83b339214d3d2
[DEBUG]   boost-utility|e395ce4635f7e28d687976d1d64941a660c87438
[DEBUG]   boost-uuid|e4f2e93dc63052cfbe3a5f90c1e92e8506c34157
[DEBUG]   boost-variant|c06a032f4afc0af7874c05fe808b3c42ecd1b86e
[DEBUG]   boost-variant2|84de20c0a5263f6fdd94d7256343e9b8aed500e9
[DEBUG]   boost-vmd|963e509355066f1f22975bb1eb5bcc77026f5263
[DEBUG]   boost-wave|239f5cf1df91181939a074ef133af5ad3468b1ed
[DEBUG]   boost-winapi|cbb8fcff6c02a2d0241498c42f1d667b6e3943ad
[DEBUG]   boost-xpressive|5277cd4354eae16f57d7b967027d09a7c19af795
[DEBUG]   boost-yap|851f71a229dbae297165a28d49992a48240fa3f8
[DEBUG]   cmake|3.20.2
[DEBUG]   features|core
[DEBUG]   portfile.cmake|c9594fbc299160fef8338ab8f2bb74f8185694f5
[DEBUG]   ports.cmake|acbdb0c75aa1c0571ad039ecdb4d4c85c14911fa
[DEBUG]   post_build_checks|2
[DEBUG]   triplet|x64-linux
[DEBUG]   triplet_abi|ff56c46e678731ac06debf9ff7c26bcff0775d07-497032558309d954763445f6b33535bb41c0820b-f537c274dd21f214ced727e33e37e05781645f01
[DEBUG]   usage|e40ab0c6357592245cd71d0f0955d6e71b69f800
[DEBUG]   user-config.jam|0089639e9c88dac7e49ba1e57f72df5837d77d02
[DEBUG]   user-config.jam|d95b4e5df155ca08f78ce582bfa423ae53b3fc2b
[DEBUG]   vcpkg.json|6d1b257f2bea5cd9bb67975cc0838d87478677a6
[DEBUG] </abientries>
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/d0/d0e14dc3de84d6cec634c822cf5c1be137105d71.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/16/16bb73509765dee82e1d30958bca378660bc4fdb.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/ff/ffbf0331ccfff2f5d79fa580d871a442c3d886c0.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/cc/cc82c971d17bf22dd8a44f68dbcee4b46c5f5bd7.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/96/963e509355066f1f22975bb1eb5bcc77026f5263.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/7f/7f2e684e9e10b7062573227612adbb5de70ab904.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/93/931432a6328946a29c9d97d0d6f92faf89eb34b2.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/d1/d10ca8bac6d50c17c352f39af91897f339c66938.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/de/de06cd55f5f3c3008d380e8e978ca4f0d5ed54db.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/da/da74a16a2c28175d0f2155233c8a9134e5474fbe.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/23/23bbc0ec8e179da2aa96e6f1aab0b13ad3b61574.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/f0/f0f8872146029e7ef5edaac4129c5a2a5d616cb4.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/ae/ae96a6214dfb68c9e1c7bda46b823cf925668f0f.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/3d/3da9a0cd1e898ad6a4dcb05025af248dd388ae95.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/fe/fec11d42c649937a9250343aca3f42a3911f44e0.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/27/27e7202ea9680bfa6bef1d4b18d6a71d844735a7.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/13/139c3cfbdbc09f2a2931130e30195d6e79a9d514.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/23/239f5cf1df91181939a074ef133af5ad3468b1ed.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/85/851f71a229dbae297165a28d49992a48240fa3f8.zip
Could not locate cached archive: /home/dominic/.cache/vcpkg/archives/8f/8f98badac5d6911fadb5dca8fa18b7751f30535a.zip
Starting package 1/20: python3:x64-linux
Building package python3[core]:x64-linux...
[DEBUG] popen(/home/dominic/dev/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/bin/cmake "-DALL_FEATURES=deprecated-win7-support;" -DCURRENT_PORT_DIR=/home/dominic/dev/vcpkg/ports/python3 -D_HOST_TRIPLET=x64-linux -DFEATURES=core -DPORT=python3 -DVCPKG_USE_HEAD_VERSION=0 -D_VCPKG_DOWNLOAD_TOOL=BUILT_IN -D_VCPKG_EDITABLE=0 -D_VCPKG_NO_DOWNLOADS=0 -DCMD=BUILD -DDOWNLOADS=/home/dominic/dev/vcpkg/downloads -DTARGET_TRIPLET=x64-linux -DTARGET_TRIPLET_FILE=/home/dominic/dev/vcpkg/triplets/x64-linux.cmake -DVCPKG_BASE_VERSION=2021-05-05 -DVCPKG_CONCURRENCY=17 -DVCPKG_PLATFORM_TOOLSET=external -DGIT=/usr/bin/git -DVCPKG_ROOT_DIR=/home/dominic/dev/vcpkg -DPACKAGES_DIR=/home/dominic/dev/vcpkg/packages -DBUILDTREES_DIR=/home/dominic/dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=/home/dominic/dev/vcpkg/installed -DDOWNLOADS=/home/dominic/dev/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P /home/dominic/dev/vcpkg/scripts/ports.cmake 2>&1)
-- Using /home/dominic/dev/vcpkg/downloads/python-cpython-v3.9.5.tar.gz
-- Cleaning sources at /home/dominic/dev/vcpkg/buildtrees/python3/src/v3.9.5-214b854b69.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/dominic/dev/vcpkg/downloads/python-cpython-v3.9.5.tar.gz
-- Applying patch 0001-static-library.patch
-- Applying patch 0002-use-vcpkg-zlib.patch
-- Applying patch 0003-devendor-external-dependencies.patch
-- Applying patch 0004-dont-copy-vcruntime.patch
-- Applying patch 0005-only-build-required-projects.patch
-- Applying patch 0006-fix-duplicate-symbols.patch
-- Using source at /home/dominic/dev/vcpkg/buildtrees/python3/src/v3.9.5-214b854b69.clean
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
-- Installing x64-linux-dbg
-- Building x64-linux-rel
-- Installing x64-linux-rel
CMake Error at ports/python3/portfile.cmake:175 (file):
  file COPY cannot find
  "/home/dominic/dev/vcpkg/packages/python3_x64-linux/bin": No such file or
  directory.
Call Stack (most recent call first):
  scripts/ports.cmake:139 (include)


[DEBUG] cmd_execute_and_stream_data() returned 256 after 65848314 us
Error: Building package python3:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: python3:x64-linux
  Vcpkg version: 2021-05-05-unknownhash

Additionally, attach any relevant sections from the log files above.
[DEBUG] /home/dominic/dev/fragment-cloud/dev/vcpkg/buildtrees/_vcpkg/src/vcpkg-tool-2021-05-05-9f849c4c43e50d1b16186ae76681c27b0c1be9d9/src/vcpkg/install.cpp(511)
[DEBUG] Exiting after 66378377 us (66377813 us)
```

--- a/ports/python3/portfile.cmake
+++ b/ports/python3/portfile.cmake
@@ -172,7 +172,6 @@ else()
     )
     vcpkg_install_make(ADD_BIN_TO_PATH INSTALL_TARGET altinstall)
 
-    file(COPY "${CURRENT_PACKAGES_DIR}/bin/" DESTINATION "${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin")
     file(COPY "${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin/" DESTINATION "${CURRENT_PACKAGES_DIR}/tools/${PORT}")
 
     # Makefiles, c files, __pycache__, and other junk.

--- a/ports/python3/vcpkg.json
+++ b/ports/python3/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "python3",
   "version-string": "3.9.5",
+  "port-version": 1,
   "description": "The Python programming language",
   "homepage": "https://github.com/python/cpython",
   "supports": "!(arm | uwp)",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5034,7 +5034,7 @@
     },
     "python3": {
       "baseline": "3.9.5",
-      "port-version": 0
+      "port-version": 1
     },
     "qca": {
       "baseline": "2.3.1",

--- a/versions/p-/python3.json
+++ b/versions/p-/python3.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "0c58aaaa7b980102912bbd8b13ba3ac227b37bd4",
+      "version-string": "3.9.5",
+      "port-version": 1
+    },
+    {
       "git-tree": "16ae1dd8eb0498d1357af3155c565035ba9a3f39",
       "version-string": "3.9.5",
       "port-version": 0

